.github/workflows/cleanup-after-pr.yml

```diff
@@ -1,37 +1,37 @@
 ---
-name: Clean After PR
+name: PR cleanup
```

Comment:
⛏️  Maybe something more descriptive here would be nice. This cleans up the ACR related to a pull request, so maybe `ACR cleanup` or `Container registry cleanup`, in reference to https://github.com/bitwarden/server/pull/3684/files#diff-341f42129383193d63609a63a006debfeab7e49af50f6f6add864ebd485295a0R2 ?